projectile.el

```diff
@@ -867,16 +867,31 @@ Files are returned as relative paths to the project root."
      ((eq vcs 'git) projectile-git-submodule-command)
      (t ""))))
 
-(defun projectile-get-all-sub-projects (project)
-  "Get all sub-projects for a given projects.
-PROJECT is base directory to start search recursively."
+(defun projectile-get-immediate-sub-projects (project)
+  "Get immediate sub-projects for a given project. Does not recurse.
```

Comment:
`M-x checkdoc`

* The first line should be just a single sentence
* the final sentence doesn't end with a `.`.
* PROJECT should appear somewhere in the docstring (params should be mentioned in all caps)